README.md

```diff
@@ -1,3 +1,79 @@
-# License
 
+# Fuel-Subgraph Setup Documentation
+
+### Prerequisites
+
+To build and run this project you need to have the following installed on your system:
+
+- Rust (latest stable) – [How to install Rust](https://www.rust-lang.org/en-US/install.html)
+    - Note that `rustfmt`, which is part of the default Rust installation, is a build-time requirement.
+- PostgreSQL – [PostgreSQL Downloads](https://www.postgresql.org/download/)
+- IPFS – [Installing IPFS](https://docs.ipfs.io/install/)
+- Profobuf Compiler - [Installing Protobuf](https://grpc.io/docs/protoc-installation/)
+
+### 1. Run IPFS
+Open the terminal and run the following commands:
+
+```bash
+ipfs init
+ipfs daemon
+```
+
+### 2. Configure PostgreSQL
+After installing PostgreSQL run the following commands in the terminal:
+
+```bash
+initdb -D .postgres -E UTF8 --locale=C
+pg_ctl -D .postgres -l logfile start
+createdb graph-node
+```
+`Note:` In case of issues, restart the database by deleting it and then starting it again
+
+### 3. Build Firehose-Extract
+Navigate to Firehose-Extract folder and run the following command in the terminal:
+
+```bash
+cargo build
+```
+
+### 4. Start Fuel-Firehose
+Ensure all arguments in `devel/fuelfire/fuelfire.yaml` are uncommented, then run the following command:
```

Comment:
Are these uncommented by default? Why would the user need to ensure this, if they have not modified those?

Reply:
It's not standard and you're right. I will delete it.